### ISSUE TITLE
DeprecationWarning: Buffer() is deprecated due to security and usability issues

### ISSUE THREAD

**Issue by miloskroulik** [s0/git-publish-subdir-action#29]
In the ouput of the action run, I see:

```
(node:3616) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
```

**Comment by yousifk** [comment]
#34

**Comment by BlackPhlox** [comment]
Might be relevant for anyone who stumbles upon this:
If you just have a branch in you repo you want to push the folder 'build' to. The you can use peaceiris' script:
```yml
    - name: Deploy
      uses: peaceiris/actions-gh-pages@v3
      with:
        github_token: ${{ secrets.GITHUB_TOKEN }}
        publish_dir: ./build
        publish_branch: master
```
Notice: This works right out of the box, you do not need to set a GITHUB_TOKEN, github does that automatically for you as opposed to provide a ssh 
private and public key.

**Comment by s0** [comment]
Fixed in #42

**Comment by s0** [comment]
Now fixed in latest release: https://github.com/s0/git-publish-subdir-action/releases/tag/v2.4.1
